doc/kinsol/guide/source/Usage/index.rst

```diff
@@ -1102,6 +1104,22 @@ negative, so a test ``retval`` :math:`<0` will catch any error.
       ``examples/kinsol/serial/kinAnalytic_fp.c``
 
 
+.. c:function:: int KINSetDampingFn(void* kin_mem, KINDampingFn damping_fn)
+
+   Sets the function used to compute the damping factor, :math:`\beta_n`, in
+   fixed-point or Picard iterations.
+
+   :param kin_mem: pointer to the KINSOL memory block.
+   :param damping_fn: the function to compute the damping parameter.
```

Comment:
Can a user remove the function by passing `NULL`?

Reply:
Yes, I'll add that to the docs